### PR TITLE
Bug #154050 fix: The filters of the list view of types are not in searchtools

### PR DESCRIPTION
**Pull request by ChavanAnkitaMahesh**

### CODE REVIEW

File: src/components/com_tjucm/administrator/models/forms/filter_types.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/models/forms/filter_types.xml:1]: @ChavanAnkitaMahesh format the XML with tab indentation

File: src/components/com_tjucm/administrator/models/forms/filter_types.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        hint="COM_TJUCM_SEARCH_FILTER_SUBMIT"
+        label="COM_USERS_FILTER_SEARCH_DESC"
+        class="js-stools-search-string" />
+        
+        <field name="state"
+               type="status"
+               label="COM_TJUCM_FILTER_STATE"
+               description="COM_TJUCM_FILTER_STATE_DESC"
+               onchange="this.form.submit();"
+        >
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/models/forms/filter_types.xml]: @ChavanAnkitaMahesh field element is not closed properly 
`>` => `/>`

File: src/components/com_tjucm/administrator/views/types/tmpl/default.php [hunk 3]
```diff
@@ -158,10 +143,10 @@
 				</div>
 				<div class="btn-group pull-right">
 					<label for="sortTable" class="element-invisible"><?php echo JText::_('JGLOBAL_SORT_BY'); ?></label>
-					<select name="sortTable" id="sortTable" class="input-medium" onchange="Joomla.orderTable()">
-						<option value=""><?php echo JText::_('JGLOBAL_SORT_BY'); ?></option>
-						<?php echo JHtml::_('select.options', $sortFields, 'value', 'text', $listOrder); ?>
-					</select>
+						<select name="sortTable" id="sortTable" class="input-medium" onchange="Joomla.orderTable()">
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/views/types/tmpl/default.php]: @ChavanAnkitaMahesh maintain indentation throughout the file

File: src/components/com_tjucm/administrator/views/types/view.html.php [hunk 4]
```diff
@@ -41,8 +41,10 @@ public function display($tpl = null)
 		$this->state = $this->get('State');
 		$this->items = $this->get('Items');
 		$this->pagination = $this->get('Pagination');
+		$this->filterForm = $this->get('FilterForm');
+        $this->activeFilters = $this->get('ActiveFilters');
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/views/types/view.html.php]: @ChavanAnkitaMahesh maintain indentation in the file

File: src/components/com_tjucm/languages/administrator/en-GB/en-GB.com_tjucm.ini [hunk 5]
```diff
@@ -176,3 +176,6 @@ COM_TJUCM_FIX_DATABASE="Fix Database"
 ; Added in version 1.2.0
 COM_TJUCM_PERMISSION_TYPE_ITEM_DELETEOWN="Delete Own Item"
 COM_TJUCM_PERMISSION_TYPE_ITEM_DELETEOWN_DESC="Allows users in the group to delete own items in this UCM type."
+
+; Added in version 1.2.4
+COM_TJUCM_SEARCH_FILTER_SUBMIT="Search"
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/languages/administrator/en-GB/en-GB.com_tjucm.ini]: @ChavanAnkitaMahesh blank line needed at the end of each language file

File: src/components/com_tjucm/administrator/models/forms/filter_types.xml (hunk 1 above)
Comment (ChavanAnkitaMahesh) [reply src/components/com_tjucm/administrator/models/forms/filter_types.xml:1]: Resolved

File: src/components/com_tjucm/administrator/views/types/tmpl/default.php (hunk 3 above)
Comment (ChavanAnkitaMahesh) [reply src/components/com_tjucm/administrator/views/types/tmpl/default.php]: Resolved

File: src/components/com_tjucm/administrator/views/types/view.html.php (hunk 4 above)
Comment (ChavanAnkitaMahesh) [reply src/components/com_tjucm/administrator/views/types/view.html.php]: Resolved

**Pull request merged**